src/routes.py

```diff
@@ -0,0 +1,282 @@
+from flask import redirect, render_template, url_for, request, flash, send_from_directory
+from src import app
+from flask_login import login_user, login_required, logout_user, current_user
+from werkzeug.security import generate_password_hash, check_password_hash
+from src.models import USER , DATA, Weekly
+from src import db
+from werkzeug.utils import secure_filename
+import uuid as uuid
+import os
+from src import ALLOWED_EXTENSIONS
+
+
+
+
+@app.route('/')
+def hello():
+    return render_template('index.html')
+
+
+@app.route("/createaccount/<int:a>", methods=["POST", "GET"])
```

Comment:
In the future with the different checks we could make a specific function for different check rather than having a nested if else if structure